apps/E2E/src/common/consts.ts

```diff
@@ -124,3 +139,5 @@ export const BASE_TESTPAGE = 'Base_TestPage';
 // ScrollViewer that contains the list of test page buttons
 export const TESTPAGE_BUTTONS_SCROLLVIEWER = 'TestPage_Buttons_ScrollViewer';
 
+// ScrollViewer that contains the list of test page buttons
+export const TESTPAGE_TESTS_SCROLLVIEWER = 'TestPage_Tests_ScrollViewer';
```

Comment:
I'd also change this to:

  `TESTPAGE_CONTENT_SCROLLVIEWER`

Better represents that this is the scrollviewer on each test page that wraps the content